samples/dashboards/demo/climate/demo.js

```diff
@@ -284,26 +286,23 @@ async function setupBoard() {
                     minRange: 30 * 24 * 3600 * 1000, // 30 days
                     maxRange: 2 * 365 * 24 * 3600 * 1000, // 2 years
                     events: {
-                        afterSetExtremes: function (e) {
-                            window.clearTimeout(selectionTimeout);
-                            selectionTimeout = window.setTimeout(async () => {
-                                if (
-                                    activeTimeRange[0] !== e.min ||
-                                    activeTimeRange[1] !== e.max
-                                ) {
-                                    activeTimeRange = [e.min, e.max];
-
-                                    const newColumn =
-                                        activeColumn[0] === 'T' ? activeColumn + activeScale : activeColumn;
-                                    await updateBoard(
-                                        board,
-                                        activeCity,
-                                        newColumn,
-                                        activeScale,
-                                        false
-                                    );
-                                }
-                            }, 50);
+                        afterSetExtremes: async function (e) {
+                            if (
+                                activeTimeRange[0] !== Math.round(e.min) ||
+                                activeTimeRange[1] !== Math.round(e.max)
+                            ) {
+                                activeTimeRange = [
+                                    Math.round(e.min),
```

Comment:
Would not be faster if the `Math.round(e.min)` were assigned to a variable? We avoid double calculations.